src/musicbrainz/tagfetcher.h

```diff
@@ -76,5 +103,9 @@ class TagFetcher : public QObject {
 
     parented_ptr<mixxx::MusicBrainzRecordingsTask> m_pMusicBrainzTask;
 
+    parented_ptr<mixxx::CoverArtArchiveTask> m_pCoverArtArchiveTask;
```

Comment:
Oh, okay I will try to name it better. In the first place, I wanted to make it with a single request, but now, we need different tasks in order to get cover arts. I named it as a service name but I think it should have been more specified. So thanks for letting me notice.

I will name it as `CoverArtArchiveLinksTask` -> Since we get multiple links with this first task (Thumbnails and actual cover art link)
I will name the second task that exists `CoverArtArchiveDisplayImagesTask` -> we need it to show the fetched thumbnails to display which cover arts availble.
I think we need another task in the future steps, in order to get a cover art with detailed quality. and it can be named as `CoverArtArchiveImageTask`-> we will get only a image to apply on user's track.

What do you think?